pkgs/development/compilers/llvm/11/clang/default.nix

```diff
@@ -39,8 +40,12 @@ let
     ] ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
       "-DLLVM_TABLEGEN_EXE=${buildLlvmTools.llvm}/bin/llvm-tblgen"
       "-DCLANG_TABLEGEN=${buildLlvmTools.libclang.dev}/bin/clang-tblgen"
+    ] ++ lib.optionals enablePolly [
+      "-DWITH_POLLY=ON"
```

Comment:
I'm building clang to check if this makes polly available. I don't think this will break anything though.

Reply:
Seems to work!